tests/single/reputation/test_erep.py

```diff
@@ -36,14 +47,15 @@ def test_paymaster_on_account_failure(w3, entrypoint_contract, manual_bundling_m
     account = deploy_contract(w3, 'TestReputationAccount', [entrypoint_contract.address], 10 ** 18)
     paymaster = deploy_and_deposit(w3, entrypoint_contract, "TestRulesPaymaster", staked=True)
     clear_reputation()
-    # set_reputation(paymaster.address, ops_seen=5, ops_included=2)
+    set_reputation(paymaster.address, ops_seen=5, ops_included=2)
     pre = get_reputation(paymaster.address)
     assert_ok(UserOperation(
         sender=account.address,
         paymaster=paymaster.address,
         paymasterVerificationGasLimit=50000,
         paymasterData=to_hex(text="nothing"),
     ).send())
+    print("== mempool=", dump_mempool())
```

Comment:
I think we should not `print` during tests because they are not really designed to provide any human-readable output except for their results